src/test/java/com/connexta/store/service/impl/StoreServiceImplTest.java

```diff
@@ -178,31 +180,43 @@ void testSuccessfulIngest(
   }
 
   @Test
-  void testAddingIrm() throws IOException {
+  void testAddMetadata() throws IOException {
     var mockInputStream = mock(InputStream.class);
     var mockResource = mock(Resource.class);
     var mockTransformWebClient = mockWebClient(mockResource);
-    var resourceUri =
+    var irmResourceUri =
         UriComponentsBuilder.fromUri(storeUrl)
             .path(StoreController.RETRIEVE_DATA_URL_TEMPLATE)
             .build(DATASET_ID, "irm");
+    var metacardResourceUri =
+        UriComponentsBuilder.fromUri(storeUrl)
+            .path(StoreController.RETRIEVE_DATA_URL_TEMPLATE)
+            .build(DATASET_ID, "metacard");
+    var fileResourceUri =
+        UriComponentsBuilder.fromUri(storeUrl)
+            .path(StoreController.RETRIEVE_DATA_URL_TEMPLATE)
+            .build(DATASET_ID, "file");
     var mockStoreService =
         new StoreServiceImpl(
             storeUrl,
             fileStorageAdaptor,
             irmStorageAdaptor,
             metacardStorageAdaptor,
-            indexDatasetClient,
+            indexClient,
             transformClient,
             taskQueue,
             mockTransformWebClient);
-    var metadataInfo = new MetadataInfo();
-    metadataInfo.setLocation(new URL("http://location"));
-    metadataInfo.setMetadataType("irm");
+    var irmMetadataInfo = new MetadataInfo();
+    irmMetadataInfo.setLocation(new URL("http://location:1234/irm"));
+    irmMetadataInfo.setMetadataType("irm");
+
```

Comment:
✏️ Can instantiate in one line
```java
new MetadataInfo().location(new URL("http://location:1234/metacard")).metadataType("metacard");
```

Reply:
This guy doesn't have a builder implementation.

Reply:
https://github.com/connexta/ion-store-api/blob/2a881be13ab0b57f39789562c17c1d5955b44343/rest/models/src/main/java/com/connexta/store/rest/models/MetadataInfo.java#L19-L22
https://github.com/connexta/ion-store-api/blob/2a881be13ab0b57f39789562c17c1d5955b44343/rest/models/src/main/java/com/connexta/store/rest/models/MetadataInfo.java#L35-L38